app/js/components/homePage/manageSettings.jsx

```diff
@@ -9,11 +9,109 @@
 import React from 'react';
 
 export default class ManageSettings extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      AppBaseURLValue: '',
+      AppFolderPathValue: '',
+      AppStoreURLValue: ''
+    };
+
+    this.onChange = this.onChange.bind(this);
+    this.onCancel = this.onCancel.bind(this);
+  }
+
+  componentWillMount() {
+    this.setState({
+      AppFolderPathValue: 'appdata\\owa',
+      AppStoreURLValue: 'http://modules.openmrs.org'
+    });
+  }
+
+  onChange(event) {
+    event.preventDefault();
+    this.setState({ [event.target.name]: event.target.value });
+  }
+  
+  onCancel() {
+    event.preventDefault();
+    this.setState({ 
+      AppBaseURLValue: '',
+      AppFolderPathValue: 'appdata\\owa',
```

Comment:
Why the backward slashes?

Reply:
@dkayiwa  It is only 1 backslash that appears, the second backslash is used to escape it. The final output is `appdata\owa`

Reply:
Why not use a forward slash?